releases/stable-branches-and-release-cycle.md

```diff
@@ -0,0 +1,79 @@
+# Producing and supporting stable releases
+
+## Summary
+
+Today the Submariner project releases one minor version a month without capacity
+for stable branches or bugfix releases. In the following proposal we define a plan
+to make this possible starting with 0.8.
+
+## Proposal
+
+### Alignment of branch -> image releases
+
+We produce currently `<image>:devel` for every master push, along with `<image>:x.y.z` for
+the releases.
+
+Changing the `master` branch name to `devel` would make this homogeneous so our image releases
+are simply `<image>:<branch-or-tag>`.
+
+### Creating one stable branch for every minor release
+
+Approaching the release of every minor or major version a new stable branch will be forked
+off `devel` branch in the form of `release-x.y` for every existing repository.
+
+Any changes related to pinning shipyard base image will be done in that branch.
+
+At that point in time, any patches which belong to the release will need to be merged in master
+and backported to `release-x.y`. For this reason, the later we branch, the less extra work
+will be necessary to produce a stable release.  
+
+### Maintaining a stable branch
+
+Any patches that are necessary for fixing bugs in a stable branch will need to be merged
```

Comment:
Until how many stable branches do we want to continue backporting?

Reply:
I’m not sure we have enough input from downstream users to know, but it would be nice to set expectations.

Reply:
We can separate the procedure from the policy, this section deals with the procedure.

Also I think if there's demand for a fix on an actively maintained branch then we'll need to cherry pick down into it (though the latter stable branches).